### ISSUE TITLE
UNable to quit program

### ISSUE THREAD

**Issue by komusoFuu** [robaho/seashore#209]
The Quit selection is dimmed on my menu. Therefore I can not close the program. What can I do?

**Comment by robaho** [comment]
That usually means there is some other sort of problem going on. Do you know what you were doing just prior? Do you know how to use the console to look for errors?

I will check for crashes through the automated reporting and see if I see anything

**Comment by komusoFuu** [comment]
> That usually means there is some other sort of problem going on. Do you know what you were doing just prior? Do you know how to use the console to look for errors?
> 
> I will check for crashes through the automated reporting and see if I see anything

I am using this on a MAc, it has been as such since I installed and first opened. Perhaps I should download and install a fresh version

**Comment by robaho** [comment]
Yes. Please completely uninstall and delete then install the latest via the App Store

**Comment by robaho** [comment]
Closing this issue as there’s been no further response.
